sentry-spring-boot-starter/pom.xml

```diff
@@ -1,55 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't reformat the files needlessly.

Reply:
Do I need to revert it or it's ok to merge with this updated formating? Because it looks like format on this file was not the same as format with other sentry projects in `sentry-java` folder, that's why I have intentionally format it to the same style. But if you need it, I can revert, no problem.

Reply:
Please if you don't mind. it's very hard to review like this.

Reply:
No, problem, I'll do it. Sorry for this.